modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy

```diff
@@ -48,7 +48,7 @@
 						{param strings: $strings /}
 					{/call}
 				{/if}
-				{if $fixedOptions}
+				{if $fixedOptions and (length($fixedOptions) > 0)}
 					{if not $readOnly and $displayValue}
```

Comment:
Hey @alinedoleron, just for a better legibility, can you agrupe this condition with parentheses?

Like: {if ($fixedOptions) and (length($fixedOptions) > 0)}

Reply:
@interaminense Cay you take a look in this issue?